libfreerdp/core/license.c

```diff
@@ -2782,12 +2823,20 @@ BOOL license_server_configure(rdpLicense* license)
 	license->ProductInfo->dwVersion = ProductVersion;
 	license->ProductInfo->pbCompanyName = (BYTE*)ConvertUtf8ToWCharAlloc(CompanyName, &len);
 	if (!license->ProductInfo->pbCompanyName || (len > UINT32_MAX / sizeof(WCHAR)))
+	{
+		WLog_ERR(TAG, "license->ProductInfo: pbCompanyName == %p || %" PRIu32 " > UINT32_MAX",
+		         license->ProductInfo->pbCompanyName, len);
 		return FALSE;
+	}
 	license->ProductInfo->cbCompanyName = (UINT32)len * sizeof(WCHAR);
 
 	license->ProductInfo->pbProductId = (BYTE*)ConvertUtf8ToWCharAlloc(ProductName, &len);
 	if (!license->ProductInfo->pbProductId || (len > UINT32_MAX / sizeof(WCHAR)))
+	{
+		WLog_ERR(TAG, "license->ProductInfo: pbProductId == %p || %" PRIu32 " > UINT32_MAX",
```

Comment:
`len` is `size_t` - `PRIu32` should be `PRIuz`

Reply:
good catch